packages/db/tests/search/test-GranuleSearch.js

```diff
@@ -135,7 +135,8 @@ test.before(async (t) => {
       last_update_date_time: !(num % 2)
         ? t.context.granuleSearchFields.lastUpdateDateTime : undefined,
       published: !!(num % 2),
-      product_volume: !(num % 5) ? Number(t.context.granuleSearchFields.productVolume) : undefined,
+      product_volume: BigInt(Math.round(Number(t.context.granuleSearchFields.productVolume)
+         * (1 / (num + 1)))).toString(),
```

Comment:
BigInt is not necessary in this example.  If it's a big int, Number(t.context.granuleSearchFields.productVolume) can't hold the value.